renovate.json

```
@@ -1,6 +1,198 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "enabled": true,
   "extends": [
-    "config:base"
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "forkProcessing": "enabled",
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "branchConcurrentLimit": 0,
+  "prConcurrentLimit": 0,
+  "prHourlyLimit": 0,
+  "rebaseStalePrs": true,
+  "platform": "github",
+  "requireConfig": "required",
+  "rebaseWhen": "conflicted",
+  "commitBodyTable": true,
+  "separateMajorMinor": false,
+  "suppressNotifications": ["prIgnoreNotification"],
+  "helm-values": {
+    "fileMatch": ["\\.*\\.ya?ml$"]
+  },
+  "kubernetes": {
+    "fileMatch": ["\\.*\\.ya?ml$"]
+  },
+  "regexManagers":[
+    {
+      "depNameTemplate": "defenseunicorns/uds-k3d",
+      "fileMatch": ["\\.*\\.ya?ml$"],
+      "matchStrings": [
+        "# renovate: datasource=github-tags depName=(?<depName>[^ ]+) versioning=(?<versioning>.*?)\n.*?(ref:) (?<currentValue>.*)"
+      ],
+      "datasourceTemplate": "github-tags",
+      "extractVersionTemplate": "^v(?<version>.*)$"
+    },
+    {
+      "fileMatch": ["\\.*\\.ya?ml$", "^\\.vscode/settings\\.json$"],
+      "matchStrings": [
+        "# renovate: datasource=github-tags depName=(?<depName>[^ ]+) versioning=(?<versioning>.*?)( extractVersion=(?<extractVersion>.*?))?\n.*?(version:|ref:) (?<currentValue>.*)",
+        "\/\/ renovate: datasource=github-tags depName=(?<depName>[^ ]+) versioning=(?<versioning>.*?)( extractVersion=(?<extractVersion>.*?))?\n.*?(https:\/\/raw.githubusercontent.com\/defenseunicorns\/zarf\/)(?<currentValue>[^\\s]+)\/"
+      ],
+      "versioningTemplate": "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}",
+      "datasourceTemplate": "github-tags"
+
+    },
+    {
+      "depNameTemplate": "defenseunicorns/uds-cli",
+      "fileMatch": ["\\.*\\.ya?ml$", "^\\.vscode/settings\\.json$"],
+      "matchStrings": [
+        "# renovate: datasource=github-tags depName=(?<depName>[^ ]+) versioning=(?<versioning>[^\n]+)\n.*?uds@(?<currentValue>.*)",
+        "\/\/ renovate: datasource=github-tags depName=(?<depName>[^ ]+) versioning=(?<versioning>.*?)( extractVersion=(?<extractVersion>.*?))?\n.*?(https:\/\/raw.githubusercontent.com\/defenseunicorns\/uds-cli\/v)(?<currentValue>[^\\s]+)\/"
+      ],
+      "datasourceTemplate": "github-tags",
+      "extractVersionTemplate": "^v(?<version>.*)$"
+    },
+    {
+      "fileMatch": ["\\.*\\.ya?ml$"],
+      "matchStrings": ["- (?<depName>.*):(?<currentValue>.*)"],
```

```suggestion
      "matchStrings": [
        "- [\"](?<depName>.*):(?<currentValue>.*)[\"]",
        "- (?<depName>.*):(?<currentValue>.*)"
      ],
```
Tested a few ways but this seemed to be the best way to handle quoted images in a `zarf.yaml`. Doing this in one match with optional quotes ends up globbing the quotes into the version and breaking that 😭

---

Still looking at this one, might have inadvertently broke something else with the helm matcher 🤔